eZ/Publish/Core/FieldType/Tests/RelationListTest.php

```diff
@@ -718,6 +734,77 @@ public function provideInvalidDataForValidate()
         ];
     }
 
+    public function testValidateNotExistingContentRelations(): void
+    {
+        $destinationContentId = (int) 'invalid';
+        $destinationContentId2 = (int) 'invalid-second';
```

Comment:
1) Both value are 0 after cast to int. You are free to choose some number, but please change name of the variable to something meaningful e.g. $invalidDestinationContentId
2) Pro-tip: You can use [metasyntactic variables](https://en.wikipedia.org/wiki/Metasyntactic_variable) for fixtures. It's not perfect but better then numerical sufixes.

Reply:
I'm aware of them being converted to `0`, this unit test is just to replicate the Public API utilization when providing a string there, which of course should always fail.